File: Programming EF Code First/chapter4/Chapter 4 VB/Fluent/Model/Lodging.vb
Namespace Model
  Public Class Lodging
	Public Property LodgingId() As Integer
	Public Property Name() As String
	Public Property Owner() As String
	Public Property IsResort() As Boolean
	Public Property MilesFromNearestAirport() As Decimal

	Public Property DestinationId() As Integer
	Public Property Destination() As Destination
	Public Property InternetSpecials() As List(Of InternetSpecial)
	Public Property PrimaryContact() As Person
	Public Property SecondaryContact() As Person
  End Class
End Namespace

File: Programming EF Code First/chapter2/Chapter2/BAGA_FluentAPI/Model/Lodging.cs
namespace Model
{
  public class Lodging
  {
    public int LodgingId { get; set; }
    public string Name { get; set; }
    public string Owner { get; set; }
    public bool IsResort { get; set; }
    public Destination Destination { get; set; }
  }
}